hydrolib/dhydamo/geometry/mesh.py

```diff
@@ -172,6 +170,8 @@ def mesh1d_add_branch(
 ) -> None:
     """Add branch to 1d mesh, from a (list of) (Multi)LineString geometry.
     The branch is discretized with the given node distance.
+    if node distance is given as inifinity, no discretization will be performed at mid point of the branch,
+    i.e. branch is treated as a pipe
 
     Args:
```

Comment:
I notice we use different kinds of doc string formats within hydrolib.
Should we have an overall convention or is it per project different?
If it is per project different it might make difficult (impossible?) to autmatically generate the API reference documentation (I am not sure about this though, maybe it does not matter.)